python/tests/lang/definitions/test_structure.py

```diff
@@ -20,14 +20,14 @@ def test_get_substructures_by_type_with_user_defined_data(self):
 
         test_string_field_entry = create_field_entry("TestStringField", "string")
         test_file_field_entry = create_field_entry("TestFileField", "file")
-        test_data_definition = create_data_definition("TestData", fields=[test_string_field_entry, test_file_field_entry])
+        test_schema_definition = create_schema_definition("TestData", fields=[test_string_field_entry, test_file_field_entry])
```

Comment:
I won't nitpick this, right now, but this name - i.e. `TestData` - reveals a possible area of improvement. Namely, when naming test definitions (or whatever else), it might be better to name them something generic enough that a change like this - i.e. `data` -> `schema` - wouldn't mean we have to go through each name and update it.

Reply:
I agree. This PR has had me stewing on some ways to combat all the strings.